pubsub/api/Pubsub.Samples/PullMessagesAsync.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_subscriber_async_pull]
```

Comment:
Same comment here with having spread tags. Multiple tags in one sample file it's OK as long as both samples are exactly the same.

Reply:
This includes `pubsub_subscriber_async_pull` and `pubsub_quickstart_subscriber`